libs/filamat/src/GLSLPostProcessor.cpp

```diff
@@ -38,6 +40,42 @@ using namespace spvtools;
 
 namespace filamat {
 
+namespace {
+
+    std::string addMaybeUnusedToVariables(const std::string& mslShader) {
+        constexpr std::string_view unusedVariables[] = {
+            "float3x3 shading_tangentToWorld;",
+            "float3 shading_position;",
+            "float3 shading_view;",
+            "float3 shading_normal;",
+            "float3 shading_geometricNormal;",
+            "float3 shading_reflected;",
+            "float shading_NoV;",
+            "float2 shading_normalizedViewportCoord;",
+            "float3 shading_clearCoatNormal;",
+            "float filament_lodBias = frameUniforms.lodBias;",
+            "float level0 = fast::clamp(floor(log2(ssSliceRadius)) - 3.0, 0.0, setup.maxLevel);",
+            "float3 bentNormal = param_8;"
```

Comment:
Hmm, this one is a bit more difficult, there's an unused local in `ssao.mat` when `COMPUTE_BENT_NORMAL` set to 0 🤔. Can we pass a `vec3(0, 0, 0)` for an `inout` parameter in GLSL?

Reply:
We can't: https://www.khronos.org/opengl/wiki/Core_Language_(GLSL)